pkg/utils/dbinstance/service.go

```diff
@@ -49,6 +49,15 @@ func Update(ins DbInstance) (map[string]string, error) {
 		return nil, ErrNotExists
 	}
 
+	state, err := ins.state()
+	if err != nil {
+		return nil, err
+	}
+
+	if (state != "RUNNABLE") && (state != "NOT_SUPPORTED") {
```

Comment:
don't know if I like this construct. open for other solutions.

Reply:
Just remove the extra parentheses and we are good.

Reply:
fixed.